docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/focalmeter/index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Kantar FocalMeter"
+date: "2023-01-16"
+sidebar_position: 12000
```

Comment:
Every time I see higher and higher numbers in reviews, looks like now we are multiplying by 1000? :D

Reply:
Not sure why we went so high 😅  I set it that high here just to fit among the other pages which have that position.